lib/Doctrine/ORM/Configuration.php

```diff
@@ -748,6 +749,60 @@ public function setCustomDatetimeFunctions(array $functions)
         }
     }
 
+    /**
+     * Sets a map of php typed fields to DBAL types for auto-completion.
+     *
+     * Keys must be FQCNs of the PHP types
+     * Values are either FQCNs of DBAL Type or their name
+     *
+     * Any previously added mappings are discarded.
+     *
+     * @param array $mappings The map of custom DQL date/time functions.
+     * @psalm-param array<class-string, class-string<Type>|string> $mappings
+     *
+     * @return void
+     */
+    public function setTypedFieldMappings(array $mappings)
```

Comment:
Please use native type declarations here and below.

```suggestion
     */
    public function setTypedFieldMappings(array $mappings): void
```